### PR TITLE
Add dummy module and integration tests

### PR DESCRIPTION
**Pull request by Andersson007**
##### SUMMARY

1. Do you folks think if it makes sense to have it here?
2. (If answer to question 1 is yes) The tests pass locally, maybe makes sense to enable them here?

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @felixfontein @mariolenz @samccann and other folks, ideas?

**Review by samccann — COMMENTED** [review]
This is good for new collection developers, but would it annoy experienced collection developers to have to delete/edit these new files?

File: plugins/modules/dummy.py [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+    description:
+      - Divisor.
+    type: int
+    required: true
+'''
+
+EXAMPLES = r'''
+- name: Divide a number by another
+  register: result
+  my_namespace.my_collection.dummy:
```
[the remark this answers is not in the document]
Comment (felixfontein) [reply plugins/modules/dummy.py:49]: If it fits to galaxy.yml, it should be `community.FIXME.dummy` :)

**Comment by felixfontein** [comment]
I'm not sure whether it is a good idea to add such stuff to this template. For a tool which creates the files on disk it is OK I think, but in a repository where you cannot force push it's kind of annoying that you have to delete these files, and they will show up forever in the Git history. Since this repository is used to instantiate new collection repos, and force push is generally disabled by default, this can be annoying for new community collection maintainers.

**Comment by russoz** [comment]
> I'm not sure whether it is a good idea to add such stuff to this template. For a tool which creates the files on disk it is OK 

Sorry if this is something widely known (to everyone but me), but: does `ansible-galaxy` or any other tool actually does that? I know `ansible-galaxy collection init` will create the basics, but do we have something like `cookiecutter` for Ansible with readily available recipes? 

Reason I'm asking is that, this dummy module is about the usual module as shown in the basic documentation for creating modules, however there would be other recipes - in community.general we could have a template for a module based on `ModuleHelper` or `StateModuleHelper` (I volunteer for writing those).

**Comment by Andersson007** [comment]
I see your point, let's close it then, thanks for the feedback:) At least we caught a couple of typos i also fixed in that forum post. The recipes at the first glance SGTM, I'll close the PR now, feel free to continue the conversation here or start an new one somewhere

**Comment by felixfontein** [comment]
A couple of years ago there was the idea of allowing to chose different templates for `ansible-galaxy collection init` (one would be the content of this repo, or at least an older version of it, it would have to be updated regularly in ansible-core). It never went anywhere though. Here it is: https://github.com/ansible/ansible/pull/80782

The devtools team might also have something that generates templates (or even full modules/plugins)? I think this here: https://github.com/ansible/ansible-creator I never used it, so no idea how well it does or whether it can be configured per collection.

**Pull request closed without merging**